.github/workflows/main_ci.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest # We could run on other OS but we would have to change the command to install libsndfile
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: [3.6, 3.8]
```

Comment:
no 3.7 ?

Reply:
We already test the 3.6 and the 3.8, I don't think it's mandatory.
This way the idea is to test min and max supported python's version

Reply:
Yes, that was the idea. Are you okay with that @RomainGoussault?

Reply:
Well it's free, so I don't see why we should not test 3.7 :)

Reply:
It's a plus 50 percent on the total computation cost so it's not really free.

Reply:
By free I meant that all the builds are done in parallel so it does not have any impact on the build time.

Reply:
Ok added